p2p/src/peers/mod.rs

```diff
@@ -92,8 +92,10 @@ impl Peers {
     /// the process of bootstrapping the network.
     pub fn current_ice_period(&self) -> i64 {
         if self.bootstrapped {
-            i64::try_from(self.ice_period.as_secs())
-                .expect("Ice period should fit in the range of u64")
+            // FIXME: recover the actual logic commented below
+            /*i64::try_from(self.ice_period.as_secs())
+            .expect("Ice period should fit in the range of u64")*/
+            3600
```

Comment:
As discussed we could put it in the default params :)

Reply:
Done!